app.js

```diff
@@ -1,8 +1,127 @@
-// import functions and grab DOM elements
+let resetButton = document.getElementById('reset-button');
 
-// let state
+let cupsDiv = document.querySelector('.cups');
 
-// set event listeners 
-  // get user input
-  // use user input to update state 
-  // update DOM to reflect the new state
+let numCupsInput = document.getElementById('num-cups-input');
+let numCupsDiv = document.getElementById('num-cups');
+
+let tryAgainButton = document.getElementById('try-again-button');
+let beginGameButton = document.getElementById('begin-game-button');
+
+let buttonsArray = document.querySelectorAll('.cup-button');
+let cupsArray = document.querySelectorAll('.cup-image');
+
+let totalDisplay = document.getElementById('total-guesses');
+let correctDisplay = document.getElementById('correct-guesses');
+let incorrectDisplay = document.getElementById('incorrect-guesses');
+
+var correctSpot;
+//initializing it here because I want to reset it every time the user clicks try again
+//At one point it stayed under the same cup until you reset the whole game but that didn't seem right
+var totalGuesses = 0;
+var correctGuesses = 0;
+
+//accepts numTimes: integer, number of cups to create for this game
+function createGame(numTimes) {
+    let i = 1;
+    //each cup is a container div and an image and button inside of it
+    //we create numTimes many cups first, then add buttons later
+    //if we add buttons inside the loop, it's harder to label them properly
+    while (i <= numTimes) {
+        createCup();
+        i++;
+    }
+    createButtons();
+    beginGameButton.classList.add('hidden');
+    numCupsDiv.classList.add('hidden');
+    //prevent user from adding more cups while they already exist
+}
```

Comment:
clever revamp of the game! I like increasing and decreasing my chances of winning a prize of nothing